checkov/terraform/checks/resource/azure/SynapseWorkspaceAdministratorLoginPasswordHidden.py

```diff
@@ -5,7 +5,7 @@
 class SynapseWorkspaceAdministratorLoginPasswordHidden(BaseResourceCheck):
     def __init__(self):
         name = "Ensure Azure Synapse Workspace administrator login password is not exposed"
-        id = "CKV_AZURE_239"
+        id = "CKV_AZURE_242"
```

Comment:
is this change intentional?

Reply:
Yes, CKV_AZURE_239 already exists, and it has caused errors